conf/metastore-site.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+   <property>
+      <name>metastore.schema.verification</name>
+      <value>false</value>
+   </property>
+   <property>
+      <name>metastore.warehouse.dir</name>
+      <value>file:///tmp</value>
+      <description></description>
+   </property>
+   <property>
+      <name>datanucleus.schema.autoCreateAll</name>
+      <value>true</value>
+   </property>
+   <property>
+      <name>metastore.task.threads.always</name>
+      <value>true</value>
```

Comment:
Don't we need this `org.apache.hadoop.hive.metastore.MaterializationsCacheCleanerTask` here rather than `true`?

Reply:
Nope. Notice this config is from official guide here: https://cwiki.apache.org/confluence/display/Hive/AdminManual+Metastore+3.0+Administration

I'm tempted that remove this config file completely to use default one.

Reply:
just tried, the default config doesn't work out of box :(